src/components/mobile/mobile-next-query-preview.vue

```diff
@@ -31,9 +32,10 @@
       </div>
       <NextQuery
         :suggestion="nextQuery"
-        class="x-button x-button--ghost x-padding--00 x-margin--left-auto x-font-weight--bold"
+        class="x-margin--left-auto x-list x-list--gap-04 x-font-weight--bold"
       >
         {{ $t('nextQueryPreview.viewResults') }}
+        <ChevronRightIcon />
       </NextQuery>
     </div>
     <SlidingPanel
```

Comment:
Probably not from this task, but there is a vertical scroll in the sliding panel